docs/bpm-user.model.md

```diff
@@ -0,0 +1,45 @@
+# Bpm User model
+
+Contains information about a Process Services user.
+
+## Details
+
+Instances of this class are returned by the methods of the
+[Bpm User service](bpm-user.service.md). It implements the
+`UserRepresentation` interface, which is defined in the
+[Alfresco JS API]h(ttps://github.com/Alfresco/alfresco-js-api/blob/master/src/alfresco-activiti-rest-api/docs/UserRepresentation.md).
+
+```ts
+class BpmUserModel implements UserRepresentation {
```

Comment:
I prefer to not have a copy of the model in the doc could be easily go out of sync